tests/test_admin.py

```diff
@@ -0,0 +1,53 @@
+from cms.test_utils.testcases import CMSTestCase
+
+from djangocms_alias.models import Alias as AliasModel, AliasContent, Category
+from djangocms_alias.utils import is_versioning_enabled
+
+
+class FiltersTestCase(CMSTestCase):
+    def setUp(self):
+        self.superuser = self.get_superuser()
+
+    def test_language_filter(self):
+        category = Category.objects.create(name='Language Filter Category')
+        alias = AliasModel.objects.create(
+            category=category,
+            position=0,
+        )
+        expected_en_content = AliasContent.objects.create(
```

Comment:
Could you replace these with factories?

Reply:
@adam-murray the entire package uses this technique, for consistency it should use what the rest does. We can look at changing the test suite in the future if required.